### PR TITLE
Updated "Create Bootable Installers" with Big Sur

### PR DESCRIPTION
**Pull request by florisvink**
Updated "Create Bootable Installers" with Big Sur. See: https://support.apple.com/en-us/HT201372

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1910,6 +1910,9 @@ sudo sysdiagnose -f ~/Desktop/
 
 #### Create Bootable Installer
 ```sh
+# macOS 11 (Big Sur)
+sudo /Applications/Install\ macOS\ Big\ Sur.app/Contents/Resources/createinstallmedia --volume /Volumes/USB
```
Comment (herrbischoff) [inline README.md:1914]: Why did you leave `--nointeraction --downloadassets` off? All other entries have them. Please verify that they still exist or do not exist, update the pull request  and add that information to the pull request description.

File: README.md [hunk 2]
```diff
@@ -2606,6 +2609,7 @@ macOS 10.12                | Sierra             | September 20, 2016 | 10.12.6 (
 macOS 10.13                | High Sierra        | September 25, 2017 | 10.13.6 (17G65) (July 9, 2018)
 macOS 10.14                | Mojave             | September 24, 2018 | 10.14.6 (18G3020) (January 28, 2020)
 macOS 10.15                | Catalina           | October 7, 2019    | 10.15.5 (19F101) (June 1, 2020)
+macOS 11.0                 | Big Sur            | February 25, 2021  | 10.2.2 (20D80) (February 25, 2021)
```
Comment (herrbischoff) [inline README.md:2612]: Thanks, but see point 3 in https://github.com/herrbischoff/awesome-macos-command-line/blob/master/contributing.md.

**Review by herrbischoff — CHANGES_REQUESTED** [review]
Please see my annotations.

File: README.md [hunk 3]
```diff
@@ -1911,7 +1911,7 @@ sudo sysdiagnose -f ~/Desktop/
 #### Create Bootable Installer
 ```sh
 # macOS 11 (Big Sur)
-sudo /Applications/Install\ macOS\ Big\ Sur.app/Contents/Resources/createinstallmedia --volume /Volumes/USB
+sudo /Applications/Install\ macOS\ Big\ Sur.app/Contents/Resources/createinstallmedia --volume /Volumes/USB --nointeraction --downloadassets
```
Comment (herrbischoff) [inline README.md:1914]: Again: have you verified that those flags do indeed work?

**Review by herrbischoff — CHANGES_REQUESTED** [review]
If you do not feel the need to reply to my questions, I will close this pull request.

File: README.md (hunk 3 above)
Comment (florisvink) [reply README.md:1914]: Yes, they so actually: 
```
➜  ~ sudo /Applications/Install\ macOS\ Big\ Sur.app/Contents/Resources/createinstallmedia --help
Usage: createinstallmedia --volume <path to volume to convert>

Arguments
--volume, A path to a volume that can be unmounted and erased to create the install media.
--nointeraction, Erase the disk pointed to by volume without prompting for confirmation.
--downloadassets, Download on-demand assets that may be required for installation.

Example: createinstallmedia --volume /Volumes/Untitled
➜  ~
```

Comment (herrbischoff) [reply README.md:1914]: Thanks for confirming. Due diligence like this keeps this list from devolving into just another collection of half-broken lazy copy-pasting.

**Pull request merged**